dashboard/modules/snapshot/tests/test_snapshot.py

```diff
@@ -14,11 +15,82 @@
 from ray._private.test_utils import (
     format_web_url,
     run_string_as_driver,
+    run_string_as_driver_nonblocking,
 )
 from ray.dashboard import dashboard
+from ray.dashboard.modules.snapshot.snapshot_head import RayActivityResponse
 from ray.dashboard.tests.conftest import *  # noqa
 
 
+def test_inactive_component_activities(call_ray_start):
```

Comment:
Can you add an unit test that breaks if the backward compatibility of `RayActivityResponse ` is broken?

Reply:
I think this would be easier to do once we add the hook for external component_activites (coming in next PR), because currently the endpoint does return the appropriate `RayActivityResponse`

Reply:
Sure let's do a follow up in the next PR!